BUILD.yaml

```diff
@@ -15,3 +15,4 @@ packages:
         - ["yarn", "--cwd", "./install/build", "compile"]
         - ["yarn", "--cwd", "./install", "compile"]
         - ["yarn", "--cwd", "./install", "download-builtin-extensions"]
+        - ["yarn", "--cwd", "./install", "svelte:gitpod-web"]
```

Comment:
Can we add this to the prepublish step of the extension build? I would say it's better to have less steps here, since this is just a part of one extension.

https://github.com/gitpod-io/openvscode-server/blob/37230ed56a61626ab916ad9285d5789db9e3e9f9/extensions/gitpod-web/package.json#L525